jsx/Panel.js

```diff
@@ -65,7 +76,11 @@ Panel.propTypes = {
   title: React.PropTypes.string
 };
 Panel.defaultProps = {
+  initCollapsed: false,
   id: 'default-panel',
-  height: '100%',
-  title: 'Selection Filter'
+  height: '100%'
 };
+
+window.Panel = Panel;
```

Comment:
Yes it is only for backwards compatibility. If there is no code that relies on this component being in the global scope we can remove this line.